warehouse/models/mart/gtfs/_mart_gtfs.yml

```diff
@@ -57,7 +45,18 @@ models:
       - name: feed_key
         description: |
           Foreign key to the `dim_schedule_feeds` table.
-      - *gtfs_dataset_key
+      - &gtfs_dataset_key
+        name: gtfs_dataset_key
+        tests:
+          - not_null
+          - relationships:
+              to: ref('dim_gtfs_datasets')
+              field: key
+              config:
+                # there are a dozen rows of SMART transit which was deleted from Airtable
+                # this will work without exception once the Airtable dim is historical;
+                # this threshold may need to increase if the backfill occurs prior to that
+                error_if: ">20"
```

Comment:
actually this may need to be adjusted? may need to be higher

Reply:
Let's discuss this as a follow up, want to get these fixes in